src/Aggregrid.js

```diff
@@ -378,7 +378,7 @@ Ext.define('Jarvus.aggregrid.Aggregrid', {
             );
         }
 
-        if (target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) { // eslint-disable-line no-cond-assign
+        if ((target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) && target.up('.jarvus-aggregrid-row')) { // eslint-disable-line no-cond-assign
```

Comment:
@nbey this doesn't seem to accomplish anything, all cells and subcells have a `.jarvus-aggregrid-row` as a parent, and this code won't run when the RollupAggregrid's new onClick override intercepts subcell clicks